### ISSUE TITLE
extensions.omnisidebar.minSidebarWidth and sidebars collapsing

### ISSUE THREAD

**Issue by harl** [Quicksaver/OmniSidebar#149]
Reducing the width of either of the two sidebars to the point where it can't be reduced any further, causes the sidebars to collapse to a 2-pixel wide line (apparently showing 2 pixels of the sidebar itself), without any handle or frame/margin left to click or drag to bring them back.
extensions.omnisidebar.minSidebarWidth is set to 30 (never modified)
Enabling or disabling the option to show a clickable browser margin has no effect at all (showing nothing to click on - it has been like that for quite some time now).

Bottom line: My sidebars are inaccessible. (There's no life between two pixels :()
